install_scripts/templates/kubernetes/yaml/rook-1-0-3-system.yml

```diff
@@ -474,11 +474,31 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        # Rook Agent toleration. Will tolerate all taints with all keys.
+        # Choose between NoSchedule, PreferNoSchedule and NoExecute:
+        - name: AGENT_TOLERATION
+          value: NoSchedule
+        # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
+        - name: AGENT_TOLERATION_KEY
+          value: node-role.kubernetes.io/master
+        # Rook Discover toleration. Will tolerate all taints with all keys.
+        # Choose between NoSchedule, PreferNoSchedule and NoExecute:
+        - name: DISCOVER_TOLERATION
+          value: NoSchedule
+        # (Optional) Rook Discover toleration key. Set this to the key of the taint you want to tolerate
+        - name: DISCOVER_TOLERATION_KEY
+          value: node-role.kubernetes.io/master
+        - name: ROOK_LOG_LEVEL
+          value: DEBUG
```

Comment:
Was it intentional to leave it at debug?

Reply:
no but maybe i should leave it while in qa on staging